helmfile/50-applications.yaml

```diff
@@ -458,6 +458,21 @@ releases:
 # prometheus-elasticsearch-exporter
 - name: prometheus-elasticsearch-exporter
   namespace: opensearch-system
+  installed: false
+  labels:
+    app: prometheus-elasticsearch-exporter
+    group: opensearch
+  chart: ./upstream/prometheus-elasticsearch-exporter
+  version: 4.11.0
+  needs:
+  - opensearch-system/opensearch-configurer
+  values:
+  - values/prometheus-elasticsearch-exporter.yaml.gotmpl
+
+  # prometheus-opensearch-exporter
+- name: prometheus-opensearch-exporter
+  namespace: opensearch-system
+  installed: true
   labels:
     app: prometheus-elasticsearch-exporter
```

Comment:
Im thinking that you could rename all "elasticsearch" to "opensearch" for this exporter. 
So the chart, label, values file.
Could you please do that aswell? :slightly_smiling_face:

Reply:
> Im thinking that you could rename all "elasticsearch" to "opensearch" for this exporter. So the chart, label, values file. Could you please do that aswell? slightly_smiling_face

Thank you Robin, I have updated the PR :)